src/compute.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
```

Comment:
Gitignore files derived from source code. It's important to never commit them as they bloat the repo.

Reply:
Just to clarify: any transpiled JS from my TypeScript files should be in .gitignore?

Reply:
No, you add a _.gitignore_ file: https://git-scm.com/docs/gitignore.

Most commonly you configure TypeScript to emit the output to a `/dist` directory instead side-by-side in `src`. Afterwards you simply put `dist/**` in the _.gitignore_ file.